specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/ExpressRouteConnectionCreate.json

```diff
@@ -0,0 +1,50 @@
+{
+    "parameters": {
+        "expressRouteGatewayName": "expressRouteGatewayName",
+        "resourceGroupName": "resourceGroupName",
+        "api-version": "2018-08-01",
+        "subscriptionId": "subid",
+        "connectionName": "connectionName",
+        "putExpressRouteConnectionParameters": {
+            "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteGateways/expressRouteGatewayName/expressRouteConnections/connectionName",
+            "name": "connectionName",
+            "properties": {
+                "routingWeight": 2,
+                "authorizationKey": "authorizationKey",
+                "expressRouteCircuitPeering": {
+                    "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteCircuits/circuitName/peerings/AzurePrivatePeering"
+                }
+            }
+        }
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "name": "connectionName",
+                "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteGateways/expressRouteGatewayName/expressRouteConnections/connectionName",
+                "properties": {
+                    "provisioningState": "Provisioned",
+                    "expressRouteCircuitPeering": {
+                        "id": "/subscriptions/subid/resourceGroups/resourceGroupName/providers/Microsoft.Network/expressRouteCircuits/circuitName/peerings/AzurePrivatePeering"
+                    },
+                    "authorizationKey": "f28e9c99-78d8-4248-a855-c54cf6beb99d",
```

Comment:
not sure if you should have a guid here. The subid seems to be substituted with "subid" string